packages/cli/src/lib/certs/certificate_details.ts

```diff
@@ -1,56 +1,35 @@
 import formatDate from './format_date'
 import {ux} from '@oclif/core'
 import {color} from '@heroku-cli/color'
+import {SniEndpoint} from '../types/sni_endpoint'
 
-type certDisplayObject = {
-  'Common Name(s)': string,
-  'Expires At': string,
-  Issuer: string,
-  'Starts At': string,
-  Subject: string,
-  'Domain(s)'?: string[]
-}
-
-export type cert = {
-  app: any,
-  domains?: string[],
-  ssl_cert: {
-    cert_domains: string,
-    expires_at: string,
-    issuer: string,
-    starts_at: string,
-    subject: string,
-    'ca_signed?': boolean
-  }
-}
-
-export const getCertificateDetails = function (cert: cert, message = 'Certificate details:') {
+export const displayCertificateDetails = function (sniEndpoint: SniEndpoint, message = 'Certificate details:') {
   const now = new Date()
-  const autoRenewsAt = new Date(cert.ssl_cert.expires_at)
+  const autoRenewsAt = new Date(sniEndpoint.ssl_cert.expires_at)
   autoRenewsAt.setMonth(autoRenewsAt.getMonth() - 1)
 
-  if (cert.app && cert.app.acm && autoRenewsAt > now) {
+  if (sniEndpoint.app && sniEndpoint.ssl_cert.acm && autoRenewsAt > now) {
```

Comment:
This tiny and almost inconsequential change revived showing the renewal date for the certificate.